source/math/FloatComplexVectorList.ooc

```diff
@@ -155,7 +163,9 @@ FloatComplexVectorList: class extends VectorList<FloatComplex> {
 		This _fastFourierTransformHelper(input, 0, input count, buffer, 0, result, 0)
 	}
 	createFFTBuffer: static func (inputSize: Int) -> This {
-		This createDefault(This _fastFourierTransformBufferSize(inputSize))
+		result := This new(This _fastFourierTransformBufferSize(inputSize))
+		result _count = inputSize
```

Comment:
`_fastFourierTransformBufferSize` returns an `Int` which is used in the constructor to set the capacity of `result`. Then you assign `_count` to a different value. Doesn't seem like a good idea...